awesomerc.lua

```diff
@@ -372,6 +372,17 @@ awful.keyboard.append_global_keybindings({
                 end
             end
         end,
+    },
+    awful.key {
+        modifiers   = { modkey },
+        keygroup    = "numpad",
+        description = "select layout directly",
+        group       = "layout",
+        on_press    = function (index)
+            local keys_to_indices = "1234567890./*-+\n"
+            local i = string.find(keys_to_indices, index)
+            awful.layout.set(awful.screen.focused().selected_tag.layouts[i])
```

Comment:
This will not universally work. There can be no selected tag and no such layout.

I don't like adding more imperative code, but we have no choice until my `activity` PR gets merged (which adds a safer way to handle this particular `nil`).

```lua
local t = awful.screen.focused().selected_tag
if t then
    t.layout = t.layouts[i] or t.layout
end
```

Reply:
Okay. I'll fix this in the same commit as the `keys_to_indices` issue.

Reply:
As I said above, I'll reorganize the commits later in a rebase.

Your code is now on commit 571ebdd7183cbf7adb6925f8cb65beb41a03d432, which replaces old commit 0873f70a078041a44aa65228388b35fe3da2be9e (whose summary also was "Add method to select a layout directly").

(Do Git or GitHub ever forget commits when they're not on any branch?)